src/app/components/entryPage/login.js

```diff
@@ -7,11 +7,13 @@ import userStore from '../../user/userStore'
 import eventStore from '../../event/eventStore'
 
 require('./../../commonStyles/input.less')
+require('../app.less')
 
 @observer
 class Login extends React.Component {
 
     componentWillMount() {
+      this.snackalert = 'helloworld'
```

Comment:
Let's just initialize this.snackalert as an empty string rather than helloworld.